content/i18n/en/landing-page.json

```
@@ -27,32 +28,31 @@
   "keyFeatures": {
     "headline": "Key features",
     "paragraph": "Kyma {spelling} is a platform for extending applications with serverless functions and microservices.",
-    "spelling": "/kee-ma/",
     "features": {
       "extensibility": {
-        "title": "Extensibility platform",
+        "title": "Serverless",
         "paragraphs": [
-          "If you are looking for cloud-native ways to extend your enterprise products, Kyma is a secure and tested environment you can rely on.",
-          "First, connect external APIs and ensure a stable event flow between the external or built-in services and your own applications. Then, start molding your own code into microservices and functions that extend your applications' logic."
+          "Kyma empowers developers to easily deploy workloads in form of functions. With Kyma you can focus on coding.",
+          "Simply write your function code, add a configuration manifest and push it to Kyma using supporting CLI or directly to git."
         ]
       },
       "runtime": {
-        "title": "Application runtime",
+        "title": "Eventing",
         "paragraphs": [
-          "Build standalone Kubernetes applications from scratch. Kyma is also an application runtime that is scalable, secure, and reliable. It allows you to develop microservices-based solutions in any language and run them on Kubernetes clusters from a cloud provider of your choice."
+          "Kyma allows you to trigger workloads (functions or microservices) via in-cluster events or external business events"
```

```suggestion
          "With Kyma, you can trigger workloads (Functions or microservices) using in-cluster events or external business events."
```

(according [style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/allow-allows) and [terminology](https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology)).